### ISSUE TITLE
Ctrl+f file preview is broken on linux

### ISSUE THREAD

**Issue by chshbh** [patrickf3139/fzf.fish#19]
I think file [`__fzf_preview_file.fish`](https://github.com/patrickf3139/fzf.fish/blob/master/functions/__fzf_preview_file.fish) do not yield appropriate output for Linux systems. 

> Here it is on Gnome Manjaro.
![image](https://user-images.githubusercontent.com/9166890/90463045-a8c9e800-e127-11ea-9a74-7878b6622493.png)

**Comment by patrickf3139** [comment]
Ah shoot, thanks for reporting.
What's your output for
```
file --verison
file-5.37
magic file from /usr/share/file/magic
```
?

**Comment by patrickf3139** [comment]
In the meantime, I would recommend switching back to version 2.
```
fisher rm patrickf3139/fzf.fish
fisher add patrickf3139/fzf.fish@v2.0
```

**Comment by chshbh** [comment]
> Ah shoot, thanks for reporting.
> What's your output for

```
file --version
file-5.39
magic file from /usr/share/file/misc/magic
seccomp support included
```

**Comment by chshbh** [comment]
I fixed the issue, but I am no longer using `file`, rather `ls`. I can send a pull request, locally I am using [exa](https://github.com/ogham/exa) which is faster than `file` or `ls` command.  IMO `ls` is fine too, since differences are in milliseconds.

**Comment by patrickf3139** [comment]
Yes please PR! Thank you

**Comment by patrickf3139** [comment]
Fixed in #21 .
